source/includes/_file_proxy_api.md.erb

```diff
@@ -0,0 +1,78 @@
+# HMDA File Serving
+
+The HMDA Platform serves a number of files including:
+
+1. Documentation
+1. Modified LAR
+1. Institution Register Summaries
+1. Dynamic National Loan-Level Dataset
+1. Snapshot National Loan-Level Dataset
+1. Rate Spread Data
+
+The HMDA Platform serves these files in two seperate ways, through the HMDA File api or direct S3 links.
```

Comment:
Some documentation is hosted on Github, though maybe the above list is referencing *other* documentation (i.e. PDFs as opposed to Markdown files) since it does say "The *HMDA Platform* serves...".  
 
Not sure if this needs to be expanded upon...?